sklearn/ensemble/base.py

```diff
@@ -77,6 +77,9 @@ def _make_estimator(self, append=True):
         estimator.set_params(**dict((p, getattr(self, p))
                                     for p in self.estimator_params))
 
+        if random_state is not None:
+            randomize_estimator(estimator, random_state)
```

Comment:
Can implement the logic here, except that `set_random_state` for testing should probably make use of this same logic, so why not make it a public util?